src/main.ts

```diff
@@ -40,11 +52,11 @@ async function run(): Promise<void> {
     const owner = github.context.repo.owner;
     const repo = payload.repository.name;
 
-    const issue_number = payload.pull_request.number;
-    const issue_title = payload.pull_request.title;
-    const headref = payload.pull_request.head.sha;
-    const baseref = payload.pull_request.base.sha;
-    const labels = payload.pull_request.labels;
+    const pr = payload.pull_request;
+    const headref = pr.head.sha;
+    const baseref = pr.base.sha;
+    const labels = pr.labels;
+    const reviewers = pr.requested_reviewers.map((r) => r.login);
```

Comment:
Not sure if this is null-safe. What happens if a PR has no reviewers?

Reply:
Good point. I've applied the [optional-chaining operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining) in combination with the [nullish-coalescing operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing) to make this null-safe.